tool/tsh/tsh.go

```diff
@@ -1058,6 +1092,26 @@ func loadIdentity(idFn string) (*client.Key, ssh.HostKeyCallback, error) {
 	if err != nil {
 		return nil, nil, trace.Wrap(err)
 	}
+	// validate TLS Cert (if present):
+	if len(tlsCert) > 0 {
+		_, err := tls.X509KeyPair(tlsCert, keyBuf.Bytes())
+		if err != nil {
+			return nil, nil, trace.Wrap(err)
+		}
+	}
+	/// Validate TLS CA certs (if present).
+	var trustedCA []auth.TrustedCerts
+	if len(tlsCACerts) > 0 {
+		var trustedCerts auth.TrustedCerts
+		pool := x509.NewCertPool()
+		for _, certPEM := range tlsCACerts {
+			if !pool.AppendCertsFromPEM(certPEM) {
+				return nil, nil, trace.BadParameter("failed to parse tls CA cert from identity file")
```

Comment:
I would think of some user action as well, as it's not clear what to do at this point